File: src/main/java/pl/yalgrin/playnite/simplesync/dto/GameIdsDTO.java
package pl.yalgrin.playnite.simplesync.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

import java.io.Serial;

@Data
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
public class GameIdsDTO extends AbstractDTO {
    @Serial
    private static final long serialVersionUID = 3080905186398003975L;

    @JsonProperty("GameId")
    private String gameId;
    @JsonProperty("PluginId")
    private String pluginId;
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/change/dto/GameIdsDTO.kt
package pl.yalgrin.playnite.simplesync.change.dto

import pl.yalgrin.playnite.simplesync.common.dto.AbstractDTO
import java.io.Serial

data class GameIdsDTO(
    var gameId: String,
    var pluginId: String
) : AbstractDTO() {
    companion object {
        @Serial
        private const val serialVersionUID: Long = 6037277647560770671L
    }
}